recipes/dsistudio/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "dsistudio" %}
+{% set version = "2021.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/frankyeh/DSI-Studio/archive/refs/tags/2021.10.05.zip
+    sha256: 6a9d1d2e249063a3bafc1fa6a99eebe776b35bcb1fd3034c24d290240be6ecfa
+    folder: src
+  - url: https://github.com/frankyeh/DSI-Studio-atlas/archive/refs/tags/2021.10.zip
+    sha256: 0ea27148512608674fe57e6eb10e523787202fb1f4de30d61d6d973e7c7f0367
+    folder: atlas
+  - url: https://github.com/frankyeh/TIPL/archive/refs/tags/2021.10.04.zip
+    sha256: b5926185484543c2067cdd1d20787e0372fd01636632fd5a87a5f358225bf765
+    folder: src/tipl
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - jom                            # [win]
+    - make                           # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - {{ cdt('mesa-dri-drivers') }}  # [unix]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('expat-devel') }}       # [linux]
+  # These packages provide links to compilers so qmake can find them easier
+    - gcc                            # [linux]
+    - gxx                            # [linux]
+  host:
+    - xorg-libxfixes                 # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - qt
+    - libglu                         # [linux]
+    - boost-cpp
+    - zlib
+  run:
+    - qt
+    - libglu                         # [linux]
+    - ucrt                           # [win]
+
+app:
+  entry: ${PREFIX}/dsi_studio.exe                 # [win]
+  entry: ${PREFIX}/dsi_studio                     # [linux]
+  entry: open ${PREFIX}/dsi_studio.app            # [osx]
+  summary: Diffusion MRI Tractography Tool
+  type: desk
+
+about:
+  home: http://dsi-studio.labsolver.org
+  license: CC BY-NC-SA 4.0 (DSI Studio, atlases), BSD (TIPL)
+  license_family: CC, BSD
```

```suggestion
  license: CC BY-NC-SA 4.0 AND BSD-2-Clause
  license_family: CC AND BSD
```
Apparently, the specification uses 'AND', 'OR', etc

---

THANKS! I will continue to work on other errors.